### PR TITLE
Delete files

### PR DESCRIPTION
**Pull request by twodarek**
Issue #29

### CODE REVIEW

File: src/printapp/document.py [hunk 1]
```diff
@@ -4,6 +4,41 @@
...
+	"""
+	document_id = ObjectId(document_id)
+	try:
+		fs = gridfs.GridFS(mongo.db, 'document_fs')
+	except TypeError as err:
+		raise DatabaseError(err)
+
+	if fs.exists(_id=document_id, email=email):
+		fs.delete(document_id)
+
```
Comment (tylervz) [inline src/printapp/document.py:21]: Use spaces instead of tabs

**Comment by tylervz** [comment]
Great work! Convert tabs to spaces and then it's all good.

File: src/printapp/document.py (hunk 1 above)
Comment (twodarek) [inline src/printapp/document.py:21]: Oops, apparently :set tabstop=4 didn't work for me...  Fixed.

**Comment by twodarek** [comment]
Thanks!  It looks like my tabstop didn't take in vim for some reason.  Fixed.

**Pull request merged**